lib/src/markdown.service.ts

```diff
@@ -423,4 +442,12 @@ export class MarkdownService {
           : line;
       }).join('\n');
   }
+
+  private useExtensions() {
+    if (this.extensions) {
+      for (const ext of this.extensions) {
+        marked.use(ext);
```

Comment:
You may want to think about using a [Marked instance](https://marked.js.org/using_advanced#instance) to prevent adding the same extensions multiple times. Some extensions will do strange things when added multiple times.

```js
import { Marked } from 'marked';

const marked = new Marked();

marked.use(...);
...
```

Reply:
done

Reply:
@UziTech instead of creating an instance of marked, we can just make sure that `marked.use(...)` is not called more than once right?